packages/utils/src/formatCurrencyAmount.ts

```diff
@@ -0,0 +1,12 @@
+export const formatCurrencyAmount = (amount: number, locale = 'en') => {
```

Comment:
I would prefer to not have this in trezor/utils package. Package should be used for some general code utils, this is specific for app and it's more formatter that util... We will create formatters package during app devepment so it could be moved there later, but it will be problem if it will be there because this is published package and removing it later will produce breaking change of package.

Reply:
utils is meant to be published, otherwise we can't use it in Connect. The linked comment refers to accidentally bumped version and jumping from 1.0.1 to 9.0.0 https://www.npmjs.com/package/@trezor/utils

Reply:
My bad, you are right!

Reply:
resolving in favor of [this comment](https://github.com/trezor/trezor-suite/pull/5810#discussion_r929984984)